src/microbe_stage/Microbe.cs

```diff
@@ -2260,7 +2291,8 @@ private void ChangeNodeParent(Microbe parent)
         var savedColony = Colony;
         Colony = null;
 
-        this.ReParent(parent);
+        GetParent().RemoveChild(this);
+        parent.AddChild(this);
```

Comment:
With `ReParent` proven to be not the problem, this should be reverted back to use it.